src/main/java/frc/robot/Constants.java

```diff
@@ -183,19 +199,30 @@ public static class TrapElvConstants {
     // P, I, D, Iz, FF
     public static final double[] BASE_PID = {36e-3, 5e-7, 1e-4, 0.0, 2e-6};
     public static final double[] SCORING_PID = {36e-3, 5e-7, 1e-4, 0.0, 2e-6};
-    public static final double[] WRIST_PID = {7, .1, .6, 0.0, 2e-6};
+    public static final double[] WRIST_PID = {3, .1, .1, 0.0};
+    public static final double[] WRIST_FF = {0, 0.29, 0.64}; // Ks, Kg, Kv
 
     // Simulation
     public static final int ELV_GEAR_RATIO = 70;
-    public static final int WRIST_GEAR_RATIO = 70;
     public static final double ELV_LIFT_MASS = 5.4; // kg
-    public static final double DRUM_RADIUS = Units.inchesToMeters(1);
     public static final double ELV_MIN_HEIGHT = Units.inchesToMeters(12);
     public static final double ELV_MAX_HEIGHT = Units.inchesToMeters(30);
+    public static final double DRUM_RADIUS = Units.inchesToMeters(1);
+
+    public static final double WRIST_MIN_ANGLE = Units.degreesToRadians(-90); // RADS
+    public static final double WRIST_MAX_ANGLE = Units.degreesToRadians(270); // RADS
+    public static final double WRIST_LENGTH = Units.inchesToMeters(11.877934);
+    public static final double WRIST_MOI = 0.3175242664; // Moment of Inertia
+    public static final double WRIST_GEAR_RATIO = 32.727272727272727272;
```

Comment:
I think this should be written as `360 / 11`, since that is more understandable number; there are 360 degrees over 11 teeth.

Reply:
Also, I believe Java does the math at compile time, so it's ultimately just as efficient to use the fraction.

Reply:
done